coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java

```diff
@@ -206,7 +205,16 @@ else if (isDisapproved(statusChangeEvent)) {
         }
     }
     
+    protected DocumentRouteHeaderValue getDocumentRouteHeaderValue() {
+        return KcServiceLocator.getService(RouteHeaderService.class).getRouteHeader(
+                this.getDocumentHeader().getWorkflowDocument().getDocumentId());
+    }
   
+    protected void setProtocolDocumentToApproveByDefault() {
+        DocumentRouteHeaderValue document = getDocumentRouteHeaderValue();
+        document.getDocumentType().getDefaultApprovePolicy().setPolicyValue(true);
+    }
```

Comment:
@leoherbie as mentioned earlier, it is only for this specific scenario where we merge the amendment/renewal details to the new document once amendment/renewal is approved and we blanket approve the new document.